src/azure-cli/azure/cli/command_modules/cosmosdb/tests/latest/test_cosmosdb_commands.py

```diff
@@ -48,16 +48,19 @@ def test_update_database_account(self, resource_group):
             'acc': self.create_random_name(prefix='cli', length=40)
         })
 
-        self.cmd('az cosmosdb create -n {acc} -g {rg} --kind MongoDB --ip-range-filter 10.10.10.10')
+        self.cmd('az cosmosdb create -n {acc} -g {rg} --kind MongoDB --ip-range-filter "20.10.10.10" --server-version 3.2 --enable-analytical-storage true --enable-free-tier false')
```

Comment:
could you also add test with multiple ip rules added?

Reply:
done